setup.cfg

```diff
@@ -7,7 +7,7 @@ current_version = 0.18.1
 
 [flake8]
 exclude = .git,__pycache__,build,dist,docs/source/conf.py
-max-line-length = 88
+max-line-length = 100
```

Comment:
It is set to 88 to conform with Black settings (cf. [Black documentation](https://github.com/psf/black/blob/main/docs/guides/using_black_with_other_tools.md#flake8)). If you want to change it to 100, then add

```
[tool.black]
line-length = 100
```

to `pyproject.toml` too (cf. [documentation on pyproject.toml](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-format)).